flows/fetch_S3_artifacts.py

```diff
@@ -0,0 +1,31 @@
+""" Fetch artifacts from S3 and save them locally """
+import os
+from prefect.logging import get_run_logger
+import utils
+
+
+def fetch_S3_artifact(path):
```

Comment:
this capitalization is weird, it should be `s3` to fit he Python snakecase standard

Reply:
Yeah, I corrected this.